src/dot.rs

```diff
@@ -2,11 +2,61 @@
 
 use std::fmt::Display;
 
-use crate::{Direction, NodeIndex, PortGraph, PortIndex, Weights};
+use crate::{Direction, Hierarchy, NodeIndex, PortGraph, PortIndex, Weights};
 
 /// Style of an edge in a dot graph. Defaults to "None".
 pub type DotEdgeStyle = Option<String>;
 
+/// Encode `PortGraph` and `Hierarchy` in dot format.
+///
+/// Calls the `nodes` and `ports` functions to get the labels for the nodes and ports.
+/// The `ports` function also returns the style of the edge connected to a port.
+///
```

Comment:
```suggestion
/// Calls the `ports` function to get the labels for the ports and the style of
/// the edges connected to them.
```

Reply:
I just copied the existing docstring for `dot_with...`

Reply:
Sorry, for some reason I though there was a duplicated line mentioning the nodes.